Riku/src/GameLogic/StateUpdate/Patch/Patch.h

```diff
@@ -15,5 +15,26 @@ class Patch
 	}
 	std::map<int,PlayerPatch> playerPatches;
 	std::map<std::pair<int,int>, TilePatch> tilePatches;
+
+	friend Patch operator+(Patch p1, const Patch& p2)
```

Comment:
w naszym przypadku raczej nie będzie to miało znaczenia, ale tak generalnie to operacje arytmetyczne chyba nie powinny modyfikować otrzymanych wartości

Reply:
Robiłem to z myślą o optymalizacji, ale zapomniałem dodać std::move na koniec. Wprowadziłem poprawkę. Teraz nie będziemy tworzyć nowego patcha za każdym razem przy używaniu operatora+, tylko powiększać starego. Nie wiem czy jest konwencja mówiąca o tym, że nie powinniśmy zmieniać argumentów operatora, ale w naszym przypadku i tak będziemy używać to tylko w sytuacji, gdy starą wersję będziemy wyrzucać